web/src/i18n/locales/en/activities/outcomes.yaml

```diff
@@ -0,0 +1,45 @@
+title: Outcomes and Metrics
+instruction:
+  detail: >-
+    Please describe how you will achieve the stated purpose of this activity in terms of Outcomes and Metrics.
+
+
+    An **Outcome** is a discrete and measurable improvement to Medicaid program management, monitoring, or administration, 
+    resulting from the delivery of IT system functionality. Outcomes should support the priorities of the Medicaid program 
+    and be directly enabled by the state’s IT project.
+
+
+    A **Metric** is a measure that would demonstrate whether a system is meeting an outcome. Metrics support both 
+    CMS-required and state-proposed outcomes, and are comprised of underlying data. In accordance with 42 CFR §433.112.
+
+  helpText: >-
+    **Example:**
+
+
+    **Outcome:** Develop and implement an MES Module to coordinate and improve public health and environmental services for children exposed to lead.
+
+
+    **Metrics:**
+
+    1. Percentage of children in Medicaid that are receiving lead screening at 12 and 24 months of age.
+
+
+    2. Percentage of children in Medicaid that receive timely public health case management and environmental services based on the Childhood Lead Poisoning Prevention Protocol for lead-exposed children.
+
+
+    3. Percentage of children in Medicaid with lead poisoning that had timely follow-up testing by providers.
+
+noDataNotice: There are no objectives and key results for this activity.
```

Comment:
"There are no outcomes and metrics for this activity" ?

Reply:
Thanks, I'll update that too.